exercises/robot-name/bonus_example.go

```diff
@@ -7,6 +7,8 @@ import (
 	"math/rand"
 )
 
+const testVersion = 1
```

Comment:
Not sure why the change to this robot-name exercise is here. Base branch isn't quite right yet maybe?